modules/imgcodecs/include/opencv2/imgcodecs.hpp

```diff
@@ -160,6 +160,30 @@ enum ImwritePAMFlags {
        IMWRITE_PAM_FORMAT_RGB_ALPHA       = 5
      };
 
+
+enum ImreadError {
+      IMREAD_OK = 0,
+      IMREAD_ERROR_FILE_NOT_FOUND,
+      IMREAD_ERROR_UNRECOGNIZED_FORMAT,
+      IMREAD_ERROR_SIZE_LIMIT_EXCEEDED,
+      IMREAD_ERROR_INVALID_HEADER,
+      IMREAD_ERROR_INVALID_DATA,
+     };
```

Comment:
indentation is 4 spaces.

Reply:
Other enum definitions in the same file have 7 spaces. Should i keep 7 spaces or change to 4 spaces?